app/helpers/number_helper.rb

```diff
@@ -0,0 +1,82 @@
+module NumberHelper
+  def number_to_human_size(number, *args)
+    options = args.extract_options!.reverse_merge(:significant => false, :precision => 1)
+    NumberHelper.handling_negatives(number) do |number|
+      super(number, options)
```

Comment:
...right @matthewd ? :sob: Figured I'd confirm since I've been failing so hard on grokking Ruby inheritance the past day.